src/views/v2/rsvp-kitchen-sink.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Block: RSVP V2 Kitchen sink
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp-kitchen-sink.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link  {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @var Tribe__Tickets__Editor__Template $this
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: adfdfe2cb0af96b96e9aa820b66bf40f</sub>